.env.example

```
@@ -108,9 +109,11 @@ LOKI_URL=http://localhost:3100
 LOKI_USERNAME=admin
 LOKI_PASSWORD=admin
 
-LOKI_URL=http://146.190.29.56:3100
-LOKI_USERNAME=admin
-LOKI_PASSWORD=admin
+# Control Panel
+PUBLIC_CONTROL_PANEL_URL=http://localhost:5173
+CONTROL_PANEL_JWT_SECRET=replace-with-a-strong-secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CONTROL_PANEL_JWT_SECRET key should go before the PUBLIC_CONTROL_PANEL_URL key

(UnorderedKey)